ci/nightly/pipeline.template.yml

```diff
@@ -1080,16 +1080,28 @@ steps:
               composition: postgres-consistency
               args: ["--seed=$BUILDKITE_JOB_ID", "--max-runtime-in-sec=1200"]
 
-      - id: output-consistency-version
-        label: "Output consistency (version)"
+      - id: output-consistency-version-dfr
```

Comment:
At some point if we split it into many different strategies, maybe run each strategy after each other, but still in the same buildkite job? There is unfortunately some overhead to setting up an agent, downloading from dockerhub, etc.